portfolio/src/main/webapp/includes.js

```diff
@@ -0,0 +1,2 @@
+
+  //includeContact();
```

Comment:
I think this whole file should be deleted.

Reply:
Deleted in PR 7